app/models/user.rb

```diff
@@ -22,12 +22,17 @@ class User < ApplicationRecord
   # TODO Which fields do we want updated each time a user logs in?
   #   - Group memberships
   def self.from_omniauth(auth)
-    User.find_or_create_by(provider: auth.provider, uid: auth.uid) do |new_user|
+    user = User.find_or_create_by(provider: auth.provider, uid: auth.uid) do |new_user|
```

Comment:
This is most certainly wrong, but I wasn't sure the best way to _only_ mutate ldap groups.